README.md

```diff
@@ -87,13 +93,13 @@ From the project root folder, fire the following commands to execute this projec
         --shutdown-time    10 \
     ```
 
-4. Authorize application on hydra
-    
+5. Authorize application on hydra
```

Comment:
@claudiosegala, now that we have an example web app, couldn’t we just remove sections 5 and 6 and just add the example app as dependency at the available docker-compose.yml file?

Reply:
@abilioesteves, isn't it better to just add the whisper-examples link, instead of replacing?

Reply:
The example we created is much better than the default one from Hydra. We'll have a docker image available for the example, making it easy for future users to execute Whisper and get a taste for its functionalities. The less cognitive overhead we have on our documentation, the better users will understand how Whisper works. Could you please do it, @claudiosegala? I'll start a docker image generation pipeline for the Web app whisper example.